src/angular.headroom.js

```diff
@@ -1,37 +1,37 @@
-(function(angular) {
-
-  if(!angular) {
-    return;
```

Comment:
Adding this back in because tests were breaking, and this is the quickest fix for now